apps/server/src/modules/files-storage/domain/filerecord-properties.factory.ts

```diff
@@ -0,0 +1,51 @@
+import { EntityId } from '@shared/domain';
+import { v4 as uuid } from 'uuid';
+import type { FileRecordParams } from '../controller/dto';
+import type { FileDto } from '../dto';
+import { IFileRecordParams, IFileSecurityCheckParams, ScanStatus } from './filerecord.do';
+
+// TODO: should be also used for fileRecord.copy actions, but dependecy cycle
+// TODO: no clean line up to entity default values atm
+// Note: can also be used for buildTask() buildDraftTask() ... chains
```

Comment:
Needs an update with filerecord examples instead of task

Reply:
I have no good idea for it. :D